### PR TITLE
Update description and keyword of the package strapi

### PR DESCRIPTION
**Pull request by bromso**
#### Description of what you did:

This PR Updates the package `strapi`'s description and keywords

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [x] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [ ] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: packages/strapi/package.json [hunk 1]
```diff
@@ -103,5 +85,48 @@
...
+    "SQLite",
+    "graphqL",
+    "infrastructure",
+    "backend",
+    "open source",
+    "self hosted",
+    "javascript",
+    "lerna",
+    "lernajs"
+  ],
```
Comment (soupette) [inline packages/strapi/package.json]: Is it possible to also add `react` in the keywords array?

File: packages/strapi/package.json [hunk 2]
```diff
@@ -103,5 +85,48 @@
...
+    "orm",
+    "realtime",
+    "rest",
+    "restful",
+    "security",
+    "jam",
+    "jamstack",
+    "javascript",
+    "headless",
+    "serverless",
```
Comment (soupette) [inline packages/strapi/package.json]: I don't think that this one can be applied

**Comment by alexandrebodin** [comment]
@bromso Are you going to continue working on this PR ?

File: packages/strapi/package.json [hunk 3]
```diff
@@ -1,26 +1,8 @@
 {
   "name": "strapi",
   "version": "3.0.0-beta.1",
-  "description": "An open source solution to create and manage your own API. It provides a powerful dashboard and features to make your life easier.",
+  "description": "An open source solution to create and manage your own API. It provides a powerful dashboard and features to make your life easier. Databases supported: MongoDB, MySQL, MariaDB, PostgreSQL, SQLite",
```
Comment (davidkartuzinski) [inline packages/strapi/package.json]: "An open source headless CMS solution to create and manage your own API. It provides a powerful dashboard and features to make your life easier. Databases supported: MongoDB, MySQL, MariaDB, PostgreSQL, SQLite"

Shouldn't we mention something about being a headless CMS?

Comment (bromso) [reply packages/strapi/package.json]: Sounds good to me. I can try to get that in my PR.

File: packages/strapi/package.json (hunk 1 above)
Comment (bromso) [reply packages/strapi/package.json]: Of course, I am on it!

**Pull request merged**